src/app/core/theme.component.ts

```diff
@@ -6,126 +6,51 @@ import {
-          <p>Large close</p>
-          <app-circular-button icon="close" size="lg"></app-circular-button>
-        </div>
-        <div class="circular-buttons-container__btn">
-          <p>Extra large close</p>
-          <app-circular-button icon="close" size="xl"></app-circular-button>
-        </div>
-        <div class="circular-buttons-container__btn">
-          <p>Small back</p>
-          <app-circular-button icon="arrow_back" size="sm"></app-circular-button>
-        </div>
-        <div class="circular-buttons-container__btn">
-          <p>Medium back</p>
-          <app-circular-button icon="arrow_back" size="md"></app-circular-button>
-        </div>
-        <div class="circular-buttons-container__btn">
-          <p>Large close</p>
-          <app-circular-button icon="arrow_back" size="lg"></app-circular-button>
-        </div>
-        <div class="circular-buttons-container__btn">
-          <p>Extra large close</p>
-          <app-circular-button icon="arrow_back" size="xl"></app-circular-button>
-        </div>
-        <div class="circular-buttons-container__btn">
-          <p>Small forward</p>
-          <app-circular-button icon="arrow_forward" size="sm"></app-circular-button>
-        </div>
-        <div class="circular-buttons-container__btn">
-          <p>Medium forward</p>
-          <app-circular-button icon="arrow_forward" size="md"></app-circular-button>
-        </div>
-        <div class="circular-buttons-container__btn">
-          <p>Large forward</p>
-          <app-circular-button icon="arrow_forward" size="lg"></app-circular-button>
-        </div>
-        <div class="circular-buttons-container__btn">
-          <p>Extra large</p>
-          <app-circular-button icon="arrow_forward" size="xl"></app-circular-button>
-        </div>
-        <div class="circular-buttons-container__btn">
-          <p>Small text</p>
-          <app-circular-button text="click" size="sm" textSize="sm"></app-circular-button>
-        </div>
-        <div class="circular-buttons-container__btn">
-          <p>Medium text</p>
-          <app-circular-button text="click" size="md" textSize="md"></app-circular-button>
-        </div>
-        <div class="circular-buttons-container__btn">
-          <p>Large text</p>
-          <app-circular-button text="click" size="lg" textSize="lg"></app-circular-button>
-        </div>
-        <div class="circular-buttons-container__btn">
-          <p>Extra large text</p>
-          <app-circular-button text="click" size="xl" textSize="xl"></app-circular-button>
-        </div>
-        <div class="circular-buttons-container__btn">
-          <p>Extra extra large text</p>
-          <app-circular-button text="click" size="xl" textSize="xl"></app-circular-button>
-        </div>
-      </div>
+
+    <div
+      style="border: 1px solid white; padding: 20px; display: flex; justify-content: space-around"
+    >
+      <p style="text-align; color: white">Circular Buttons</p>
```

Comment:
Jeżeli dopuszczamy używanie styli inline to należy określić w jakich przypadkach można tego używać.